examples/asrc_demo/asrc_demo.cmake

```diff
@@ -0,0 +1,51 @@
+#**********************
+# Gather Sources
+#**********************
+file(GLOB APP_SOURCES ${CMAKE_CURRENT_LIST_DIR}/src/*.c ${CMAKE_CURRENT_LIST_DIR}/src/usb/*.c ${CMAKE_CURRENT_LIST_DIR}/src/gpio_test/*.c)
+set(APP_INCLUDES
+    ${CMAKE_CURRENT_LIST_DIR}/src
+    ${CMAKE_CURRENT_LIST_DIR}/src/usb
+)
+
+include(${CMAKE_CURRENT_LIST_DIR}/bsp_config/bsp_config.cmake)
+
+#**********************
+# Flags
+#**********************
+set(APP_COMPILER_FLAGS
+    -Os
+    -g
+    -report
+    -fxscope
+    -mcmodel=large
+    -Wno-xcore-fptrgroup
+    ${CMAKE_CURRENT_LIST_DIR}/src/config.xscope
+)
+
+set(APP_COMPILE_DEFINITIONS
+    DEBUG_PRINT_ENABLE=1
+    PLATFORM_USES_TILE_0=1
+    PLATFORM_USES_TILE_1=1
+    XUD_CORE_CLOCK=600
+
+    CFG_TUSB_DEBUG_PRINTF=rtos_printf
+    CFG_TUSB_DEBUG=0
+)
+
+set(APP_LINK_OPTIONS
+    -lquadspi
+    -report
+    ${CMAKE_CURRENT_LIST_DIR}/src/config.xscope
+)
+
+set(APP_COMMON_LINK_LIBRARIES
+    rtos::freertos_usb
+    rtos::drivers::my_i2s
```

Comment:
Wondering why it is my_i2s?

Reply:
would be better to give it a name that makes it's purpose more clear

Reply:
Yes! my_i2s sounds weirdly territorial when it wasn't meant to be!